invenio_cli/commands/requirements.py

```diff
@@ -196,7 +196,10 @@ def check_pipenv_installed(cls):
     @classmethod
     def check_dev(cls):
         """Steps to check the development pre-requisites."""
-        if rdm_version()[0] >= 11:
+        if rdm_version()[0] >= 12:
```

Comment:
Honestly we should revisit this afterwards since this has a cost to maintain/gets out of sync.

Reply:
i thought the same. Since we thought about v12 is the next LTS we could skip the v9 support?

Reply:
We support v9 until Feb 2025, so we can remove it then.